composable/chain.json

```diff
@@ -2,12 +2,12 @@
   "$schema": "../chain.schema.json",
   "chain_name": "composable",
   "chain_id": "centauri-1",
-  "website": "https://www.composable.finance/",
-  "pretty_name": "Composable",
+  "website": "https://www.picasso.xyz/",
+  "pretty_name": "Picasso",
   "status": "live",
   "network_type": "mainnet",
-  "bech32_prefix": "centauri",
-  "daemon_name": "centaurid",
+  "bech32_prefix": "pica",
```

Comment:
Please register this prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
registered, please lmk if the PR needs further changes, the chain upgrade is scheduled for wednesday